WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowSettingsViewController.swift

```diff
@@ -271,7 +271,11 @@ class BloggingRemindersFlowSettingsViewController: UIViewController {
         return CalendarDayToggleButton(
             weekday: weekday,
             dayName: self.calendar.shortWeekdaySymbols[localizedDayIndex].uppercased(),
-            isSelected: isSelected) { button in
+            isSelected: isSelected) { [weak self] button in
+
+            guard let self = self else {
+                return
+            }
 
             if button.isSelected {
                 self.weekdays.append(button.weekday)
```

Comment:
We could just do `self?.weekdays...` in this if / else instead of the guard, but up to you :)

Reply:
I usually prefer the guard because really none of the code below needs to run if `self == nil`.